coreos-4/policies/AC-Access_Control/component.yaml

```diff
@@ -96,21 +135,30 @@
 - control_key: AC-2 (2)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
```

Comment:
@shawndwells why this change? Local user account handling hasn't been resolved for CoreOS.